app/views/admin/audit_logs/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Audit Logs</h1>
+
+<%= render 'search_form' %>
+<br/>
```

Comment:
Why a break here after a `<div class="row">`?

Reply:
Without the break, the table sticks to the search button. This is an easy way to add a margin.